### PR TITLE
rearranged some stuff in the emails

### PR DESCRIPTION
**Pull request by mlgiardina**

### CODE REVIEW

**Comment by jah2488** [comment]
Could we format the start and end time?

**Comment by mlgiardina** [comment]
yes definitely. will do that now.

File: app/views/meeting_mailer/invited_to_meeting.html.erb [hunk 1]
```diff
@@ -6,8 +6,8 @@
     <li>Agenda: <%= @meeting.agenda %> </li>
     <li>Room: <%= @meeting.room.name %> (<%= @meeting.room.room_number %>) </li>
     <li>Location: <%= @meeting.room.location %> </li>
-    <li>Start Time: <%= @meeting.start_time %> </li>
-    <li>End Time: <%= @meeting.end_time %> </li>
+    <li>Start Time: <%= @meeting.start_time.strftime("%B %-d, %Y at %-I:%M%P") %> </li>
+    <li>End Time: <%= @meeting.end_time.strftime("%B %-d, %Y at %-I:%M%P") %> </li>
```
Comment (mlgiardina) [inline app/views/meeting_mailer/invited_to_meeting.html.erb:10]: @jah2488 how does that look?

Comment (JMD1986) [inline app/views/meeting_mailer/invited_to_meeting.html.erb:10]: good call

**Comment by JMD1986** [comment]
adding strftime to lots of stuff? cool.

we seem to be using that srftime method a lot. maybe we can write a method for it at some point.

**Comment by cecyc** [comment]
@JMD1986 that's what the class 'timeDisplay' was supposed to do ;) :P

**Pull request closed without merging**